src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age ? (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    ) : ''}
+    {!person.isMarried ? (
```

Comment:
it is bad practice to use nested ternary operator
check it
https://eslint.org/docs/latest/rules/no-nested-ternary